example/timer/odp_timer_accuracy.c

```diff
@@ -297,11 +299,15 @@ static int start_timers(test_global_t *test_global)
 		return -1;
 	}
 
-	if (timer_capa.max_timers &&
-	    test_global->alloc_timers > timer_capa.max_timers) {
+	max_timers = timer_capa.max_timers;
+
+	if (mode == 3)
+		max_timers = timer_capa.periodic.max_timers;
```

Comment:
To follow API strictly `odp_timer_capability_t.periodic.max_pools` should be checked.